operators/pkg/tenant-controller/tenant_controller_test.go

```diff
@@ -398,4 +439,20 @@ func checkTnClusterResourceCreation(ctx context.Context, tnName, nsName string,
 	Expect(createdNcSecret.Data).Should(HaveKeyWithValue("username", []byte("keycloak-"+tnName)))
 	Expect(createdNcSecret.Data).Should(HaveKey("password"))
 	Expect(createdNcSecret.Data["password"]).Should(HaveLen(40))
+
+	checkTnPVCSecretCreation(ctx, tnName, nsName, timeout, interval)
+
+	By("By checking that the NFS pvc secret of the tenant has been created")
+	NFSSecretLookupKey := types.NamespacedName{Name: NFSSecretName, Namespace: nsName}
+	createdNFSSecret := &v1.Secret{}
+	doesEventuallyExists(ctx, NFSSecretLookupKey, createdNFSSecret, BeTrue(), timeout, interval)
+
+	By("By checking that the NFS pvc secret of the tenant has a owner reference and label pointing to the tenant operator")
+	Expect(createdNFSSecret.OwnerReferences).Should(ContainElement(MatchFields(IgnoreExtras, Fields{"Name": Equal(tnName)})))
+	Expect(createdNFSSecret.Labels).Should(HaveKeyWithValue("crownlabs.polito.it/managed-by", "tenant"))
+
+	By("By checking that the NFS pvc secret of the tenant has a correct spec")
+	Expect(createdNFSSecret.Type).Should(Equal(v1.SecretTypeOpaque))
+	Expect(createdNFSSecret.Data).Should(HaveKey(NFSSecretServerNameKey))
+	Expect(createdNFSSecret.Data).Should(HaveKey(NFSSecretPathKey))
```

Comment:
Nit. You could check that also the value is correct.

Reply:
Done